manifests/ops-files/windows/use-overlay.yml

```diff
@@ -0,0 +1,7 @@
+---
+- type: replace
+  path: /instance_groups/name=worker/jobs/name=flanneld/properties?/vni
+  value: 4096
+- type: replace
+  path: /instance_groups/name=worker/jobs/name=flanneld/properties?/port
+  value: 4789
```

Comment:
Should these values also be applied to the master instance_group? What about the windows-worker group?

Reply:
We applied these properties to the masters and that fixed our metrics server issue.

Reply:
It might also make sense to rename this ops-file. We normally use overlay, this is just changing the flanneld properties that windows needs.

Reply:
Updated it to be called `overlay-overrides.yml` and included the master instance group. We didn't bother adding `etcdctl` to the Windows job, so it assumes that the configuration has already been written to etcd by one of the Linux VMs. This is a pretty safe guarantee, since the masters run flannel and will come up before the Windows node.